packages/eclipse/src/styles/globals.css

```
@@ -75,7 +82,7 @@
     --font-family-display: var(--font-mona-sans), Inter, sans-serif;
     --font-family-sans-display: Inter, sans-serif;
     --font-family-sans: Inter, system-ui, sans-serif;
-    --font-family-mono: Jetbrains Mono, monospace;
+    --font-family-mono: Monaspace Neon Var;
```

Stylelint (17.3.0)

[error] Expected "Monaspace" to be "monaspace" (value-keyword-case)

(value-keyword-case)